DOCS/man/options.rst

```diff
@@ -4442,6 +4443,33 @@ Screenshot
     Set the JPEG XL compression effort. Higher effort (usually) means better
     compression, but takes more CPU time. The default is 4.
 
+``--screenshot-avif-encoder=<encoder>``
+    Specify the AV1 encoder to be used by libavcodec for encoding avif
+    screenshots.
+    
+    Default: ``libaom-av1``
+
+``--screenshot-avif-format=<format>``
+    Specify the pixel format to the libavcodec encoder. 
```

Comment:
You should name this option "pixfmt" rather than "format" as in most cases format implies container format.